### PR TITLE
Added Manhattan Distance

### LINKED ISSUE #1377: Manhattan Distance in C, C++, Java, Python, Go
@rishabhgarg25699 @marwahmanbir please assign it to me.

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #1377 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added Manhattan Distance in 5 languages

#### Languages Used:
- C
- C++
- Python
- Go
- Java

#### Files Added:
- Manhattan_Distance.c
- Manhattan_Distance.cpp
- Manhattan_Distance.py
- Manhattan_Distance.go
- Manhattan_Distance.java

### CODE REVIEW

File: Manhattan_Distance/Manhattan_Distance.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    scanf("%d", &n);
+    int x[n];
+    int y[n];
+    for (int i = 0; i < n; i++) {
+        scanf("%d", &x[i]);
+    }
+    for (int i = 0; i < n; i++) {
+        scanf("%d", &y[i]);
+    }
+    printf("%d", ManhattanSum(x,y,n));
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.c]: Add space after commas.

File: Manhattan_Distance/Manhattan_Distance.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        for (int j = i + 1; j < n; j++) {
+            ans = ans + Manhattan(x, y, i, j);
+        }
+    }
+    return ans;
+}
+
+int main() {
+    int n;
+    cin >> n;
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.cpp]: No need to add space around input & output operators.
It's not mentioned in coding conventions & the compiler simply ignores them. Also won't make much difference in readability.

File: Manhattan_Distance/Manhattan_Distance.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import (
+    "fmt"
+    "math"
+)
+
+func Manhattan(x []int, y []int, i int, j int) int {
+    return int((math.Abs(float64(x[i] - x[j])) + math.Abs(float64(y[i] - y[j]))))
+}
+
+func ManhattanSum(x []int, y []int, n int) int{
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.go]: Space before ``{``.

File: Manhattan_Distance/Manhattan_Distance.go [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        var temp int
+        fmt.Scan(&temp)
+        x = append(x, temp)
+    }
+    for i := 0; i < n; i++ {
+        var temp int
+        fmt.Scan(&temp)
+        y = append(y, temp)
+    }
+    fmt.Print(ManhattanSum(x,y,n))
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.go]: Space after commas.

File: Manhattan_Distance/Manhattan_Distance.cpp [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    int x[n];
+    int y[n];
+    for (int i = 0; i < n; i++) {
+        cin >> x[i];
+    }
+    for (int i = 0; i < n; i++) {
+        cin >> y[i];
+    }
+    cout << ManhattanSum(x,y,n);
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.cpp]: Space after commas.

File: Manhattan_Distance/Manhattan_Distance.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        n = scan.nextInt();
+        int x[] = new int[n];
+        int y[] = new int[n];
+        for (int i = 0; i < n; i++) {
+            x[i] = scan.nextInt();
+        }
+        for (int i = 0; i < n; i++) {
+            y[i] = scan.nextInt();
+        }
+        System.out.print(ManhattanSum(x,y,n));
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.java]: Fix spacing

File: Manhattan_Distance/Manhattan_Distance.py [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+n = int(input())
+x = []
+y = []
+for i in range(0, n):
+    x.append(int(input()))
+for i in range(0, n):
+    y.append(int(input()))
+
+print(ManhattanSum(x,y,n))
```
Comment (marwahmanbir) [inline Manhattan_Distance/Manhattan_Distance.py]: Fix spacing.

**Comment by MastersAbh** [comment]
Hi @marwahmanbir , I talked to @jainaman224 regarding spaces around << and >>, 
He said there must be spaces.
in coding conventions, it is already mentioned that there must be spaces around operators and << and >> are operators. so, leaving it for now.

**Comment by MastersAbh** [comment]
Done. Please review @marwahmanbir @jainaman224

**Pull request merged**